File: src/MUA/semester1/s2/Team.java
package MUA.semester1.s2;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Team {

    public static boolean canImplementSolution(String[] ps){

        int c = 0;

        for (String p: ps) {
            if (Integer.parseInt(p) == 1){
                c ++;
            }
            if (c == 2){
                return true;
            }
        }
        return false;
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(br.readLine());
        int c = 0;

        for (int i = 0; i < n; i++) {
            String ps = br.readLine();
            c += (canImplementSolution(ps.split(" ")) ? 1 : 0);
        }
        System.out.println(c);
    }

}

File: src/codeforces/juniorTrainingRoadmap/CFA/Team.kt
package codeforces.juniorTrainingRoadmap.CFA

import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    val br = BufferedReader(InputStreamReader(System.`in`))
    val n = br.readLine().toInt()
    var count = 0
    for (i in 1 .. n) {
        val certain = br.readLine().split(" ").map { it.toInt() }.sum()
        count += if (certain >= 2 ) 1 else 0
    }
    println(count)
}